lib/lf/uscalfe/loc_comp_ellbvp.h

```diff
@@ -113,8 +113,8 @@ class ReactionDiffusionElementMatrixProvider {
    * exactness as the polynomial degree of the finite element space.
    */
   ReactionDiffusionElementMatrixProvider(
```

Comment:
This should not be changed, because the implementation heavily relies on precomputing information on the reference element(s), which is not possible for a generic `ScalarFESpace`.